assets/stylesheets/blog.scss

```diff
@@ -99,3 +99,7 @@
     width: 200px
   }
 }
+
```

Comment:
I think this comma thing is a little too much of a hack, so I would remove it 😅 and just hope we can figure out how to fix it in the future. I also don't think we have the use case very often where a blog post will belong to more than two categories. I really appreciate the effort, Chris!